src/client/Microsoft.Identity.Client/ApiConfig/AbstractPublicClientAcquireTokenParameterBuilder.cs

```diff
@@ -21,6 +23,23 @@ internal AbstractPublicClientAcquireTokenParameterBuilder(IPublicClientApplicati
             PublicClientApplicationExecutor = publicClientApplicationExecutor;
         }
 
+        public AbstractPublicClientAcquireTokenParameterBuilder<T> WithPoPAuthenticationScheme(Uri uri) // TODO: naming and comments
```

Comment:
**Proposal for XML comment**
summary: Modifies the token acquisition request so that the acquired token is a Proof of Possession token (PoP), rather than a bearer token. A key/pair is generated by MSAL.NET. See https://aka.ms/msal-net-pop
param: uri: URI for which to request a Proof of Possession token is aquired
See also: AuthorizationResult.CreateAuthorizationHeader